src/main/java/com/databasepreservation/modules/siard/path/PathStrategySIARD1.java

```diff
@@ -0,0 +1,171 @@
+package com.databasepreservation.modules.siard.path;
+
+/**
+ * Defines a SIARD 1.0 implementation to get paths to folders and files
+ * @author Bruno Ferreira <bferreira@keep.pt>
+ */
+public class PathStrategySIARD1 implements PathStrategy{
+	// names for directories
+	private static final String HEADER_DIR = "header";
+	private static final String CONTENT_DIR = "content";
+	private static final String METADATA_DIR = "metadata";
+	private static final String SCHEMA_DIR = "schema";
+	private static final String TABLE_DIR = "table";
+	private static final String LOB_DIR = "lob";
+
+	// names for files
+	private static final String METADATA_FILENAME = "metadata";
+	private static final String SCHEMA_FILENAME = "schema";
+	private static final String TABLE_FILENAME = "table";
+	private static final String LOB_FILENAME = "record";
+
+	// extensions for files
+	private static final String XML_EXTENSION = "xml";
+	private static final String XSD_EXTENSION = "xsd";
+	private static final String CLOB_EXTENSION = "txt";
+	private static final String BLOB_EXTENSION = "bin";
+
+	// control characters
+	private static final String SLASH = "/";
```

Comment:
Slash should be substituted by File.SEPARATOR and DOT should have a more semantic name, like FILENAME_EXTENSION_SEPARATOR.